Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -300,6 +300,9 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     m_extensionController = nil;
     m_contentScriptWorld = nullptr;
 
+    m_tabMap.clear();
+    m_windowMap.clear();
```

Comment:
m_windowOrderVector, m_focusedWindowIdentifier, m_extensionPageTabMap, m_actionWindowMap, m_actionTabMap, m_defaultAction, m_popupPageActionMap, should be cleared too.

m_ports, m_portQueuedMessages, m_nativePortMap for completeness (though unrelated to windows and tabs)

Reply:
I'll clear these.